pages/osx/lab.md

```diff
@@ -0,0 +1,11 @@
+# lab
+
+> A CLI for Gitlab.
+
+- Create a merge request on origin master:
+
+`lab merge-request`
+
+- Same as above but specify remote and target branch:
+
+`lab merge-request origin develop`
```

Comment:
I see this in the documentation `-b feature/feature-name -t develop`. Are the `-b` and `-t` flags not needed ?

Reply:
Hi Agnivade. Thanks for the feedback I made some update :)

No the flag are not needed. I forgot to mention that a [PR on lab](https://github.com/zaquestion/lab/pull/128/files) must be merged before this doc is relevant (there is a bug atm). Shall I close this PR for now and reopen later ?